From 0639f0c4530349e4b42ea9650686f5e7d5ac7a5d Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 22 Sep 2021 19:25:05 +0300
Subject: [PATCH] [SW-55215] gaudi: remove needless define

The spsram_init_done is only 1 bit wide.. it can either be set or clear,
so there is really no need for a define here which only obscures the
fact.
Remove the needless define.

Change-Id: I5646e60be35f4cbb4ff1d2226391fc0a5edd7d96
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi/arm/common.c b/arch/gaudi/arm/common.c
index 55f145b..c836984 100644
--- a/arch/gaudi/arm/common.c
+++ b/arch/gaudi/arm/common.c
@@ -500,7 +500,6 @@
 }
 
 #define RESET_PATTERN		0x5a5a5a5a
-#define SPSRAM_INIT_DONE	1
 
 void preboot_sp_sram_init(void)
 {
@@ -509,12 +508,12 @@
 	u32 *data = (u32 *)(HL_SP_SRAM_BASE);
 
 	cold_rst_data.data = regmap_readl(mmCOLD_RST_DATA);
-	if (cold_rst_data.spsram_init_done != SPSRAM_INIT_DONE) {
+	if (!cold_rst_data.spsram_init_done) {
 		/* SP SRAM access is limited to 32bit */
 		for (i = 0; i < HL_SP_SRAM_SIZE / sizeof(u32); i++)
 			data[i] = RESET_PATTERN;
 
-		cold_rst_data.spsram_init_done = SPSRAM_INIT_DONE;
+		cold_rst_data.spsram_init_done = 1;
 		regmap_writel(cold_rst_data.data, mmCOLD_RST_DATA);
 	}
 
